Kitodo-LongTimePreservationValidationModule/src/main/java/org/kitodo/longtimepreservationvalidationmodule/LongTimePreservationValidationModule.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.longtimepreservationvalidationmodule;
+
+import edu.harvard.hul.ois.jhove.App;
+import edu.harvard.hul.ois.jhove.JhoveBase;
+
+import java.net.URI;
+
+import org.kitodo.api.validation.ValidationResult;
+import org.kitodo.api.validation.longtimepreservation.FileType;
+import org.kitodo.api.validation.longtimepreservation.LongTimePreservationValidationInterface;
+import org.kitodo.config.Config;
+
+/**
+ * An LongTimePreservationValidationInterface implementation using Jhove.
+ */
+public class LongTimePreservationValidationModule implements LongTimePreservationValidationInterface {
+    /**
+     * Path to the {@code jhove.conf} file
+     *
+     * <p>
+     * Example:
+     * LongTimePreservationValidationModule.jhoveConf=/var/lib/tomcat8/webapps/kitodo-production/WEB-INF/classes/jhove.conf
+     */
+    private static final String PARAMETER_JHOVE_CONF = "LongTimePreservationValidationModule.jhoveConf";
+
+    /**
+     * Returns the matching module name for the given file type.
+     *
+     * @param fileType
+     *            file type that a module name shall be returned for
+     * @return the matching module name
+     */
+    private String determineModuleName(FileType fileType) {
+        switch (fileType) {
+            case GIF:
+                return "GIF-hul";
+            case JPEG:
+                return "JPEG-hul";
+            case JPEG_2000:
+                return "JPEG2000-hul";
+            case PDF:
+                return "PDF-hul";
+            case PNG:
+                return "PNG-gdm";
+            case TIFF:
+                return "TIFF-hul";
+            default:
+                throw new IllegalStateException("Complete switch");
```

Comment:
I think this is thrown exception is wrong as in this case a not supported module is called. At least get out a proper exception message as `complete switch` is not realy help- or meaningful.

Reply:
The thing is that you *cannot* call any other than the above cases because the enum `FileType` does not have any more constants. Howeyer, the compiler will complain about a missing return statement if I do nothing.